app/femr/ui/controllers/HistoryController.java

```diff
@@ -102,7 +102,13 @@ public Result indexEncounterGet(int encounterId) {
         IndexEncounterMedicalViewModel indexEncounterMedicalViewModel = new IndexEncounterMedicalViewModel();
         IndexEncounterPharmacyViewModel indexEncounterPharmacyViewModel = new IndexEncounterPharmacyViewModel();
 
+        /* Alaa Serhan */
+        ServiceResponse<SettingItem> response = searchService.retrieveSystemSettings();
```

Comment:
We still need the settings in the view to output the text and display proper units when it Is imperial or metric. If you have an idea on how we can do the formatting without it I'm ready to change it.

The only thing I can do is add a variable to models..

But that's just moving stuff around.

Not really a solution...

Let me know if you have any suggestions.